### PR TITLE
docs(ru): ru translation update

### PR DESCRIPTION
**Pull request by Yevhenii-Bakhmat**
- docs(ru): folder-structure update
- docs(ru): faq update
- docs(ru): t3-collection update

Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

* updated folder_structure.mdx
* updated t3-collection links
* updated faq links

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Comment by Yevhenii-Bakhmat** [comment]
@ronanru @AmadeusTwi

**Review by ronanru — APPROVED** [review]

File: www/src/pages/ru/faq.md [hunk 1]
```diff
@@ -49,7 +50,7 @@ lang: ru
 
 ## У меня не получается добавить i18n в мое приложение. Есть ли какие-либо рекомендации?
 
-Мы решили не включать i18n по умолчанию в `create-t3-app`, потому что это очень субъективная тема, и есть много способов ее реализации.
+We have decided against including i18n by default in `create-t3-app` because it's a very opinionated topic and there are many ways to implement it.
```
Comment (juliusmarminge) [inline www/src/pages/ru/faq.md:53]: Why English?

Comment (Yevhenii-Bakhmat) [reply www/src/pages/ru/faq.md:53]: Hmmmmmm

**Review by juliusmarminge — APPROVED** [review]
Great!

**Comment by AmadeusTwi** [comment]
Oh that happened faster than I looked :D

**Pull request merged**